frontend/src/pages/ProgressionChecker/Dashboard/Dashboard.jsx

```diff
@@ -1,15 +1,15 @@
 import React from "react";
+import { ArrowDownOutlined } from "@ant-design/icons";
 import { animated, useSpring } from "@react-spring/web";
-import { Typography } from "antd";
+import { Button, Typography } from "antd";
 import LiquidProgressChart from "components/LiquidProgressChart";
 import DegreeCard from "../DegreeCard";
 import SkeletonDashboard from "./SkeletonDashboard";
 import "./index.less";
 
-const Dashboard = ({ isLoading, degree }) => {
+const Dashboard = ({ isLoading, degree, clickArrow }) => {
```

Comment:
Is there a reason why `clickArrow` is passed down as a prop rather than being defined within the `Dashboard` component?

Reply:
The reason I passed the ```clickArrow``` function as a prop is I have defined the function in the ```ProgressionChecker.jsx``` to scroll to the ```<Divider id="divider" />``` which is in ```ProgressionChecker.jsx```.